### PR TITLE
Add amqp example

### PR DESCRIPTION
**Pull request by mnovak1**

### CODE REVIEW

File: microprofile-reactive-messaging-amqp-connector-example/scripts/generate-self-singned-certs.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Generate broker's key pair using obtained OpenShift cluster domain for `apps`
```
Comment (jmesnil) [inline microprofile-reactive-messaging-amqp-connector-example/scripts/generate-self-singned-certs.sh:1]: there's a typo in the name, it should be `generate-self-signed-certs.sh`

File: microprofile-reactive-messaging-amqp-connector-example/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Introduction
+
+This is example application used in [Wildfly Guide](https://www.wildfly.org/guides/): Use MicroProfile Reactive Messaging with AMQP Connector with SSL Connection to AMQ 7 on OpenShift
```
[the remark this answers is not in the document]
Comment (mnovak1) [reply microprofile-reactive-messaging-amqp-connector-example/README.md:3]: Yes, I'll add it.

File: microprofile-reactive-messaging-amqp-connector-example/scripts/generate-self-singned-certs.sh (hunk 1 above)
Comment (mnovak1) [reply microprofile-reactive-messaging-amqp-connector-example/scripts/generate-self-singned-certs.sh:1]: Fixed.

File: microprofile-reactive-messaging-amqp-connector-example/src/main/java/org/wildfly/examples/microprofile/reactive/messaging/amqp/connector/TestResource.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ */
+
+package org.wildfly.examples.microprofile.reactive.messaging.amqp.connector;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+
+@Path("/last")
+public class TestResource {
```
[the remark this answers is not in the document]
Comment (mnovak1) [reply microprofile-reactive-messaging-amqp-connector-example/src/main/java/org/wildfly/examples/microprofile/reactive/messaging/amqp/connector/TestResource.java:13]: I'll remove this class as in the end I don't mention it in the guide.

File: microprofile-reactive-messaging-amqp-connector-example/src/main/resources/META-INF/microprofile-config.properties [hunk 4]
```diff
@@ -0,0 +1,16 @@
+#
+# Copyright The WildFly Authors
+# SPDX-License-Identifier: Apache-2.0
+#
+
+amqp-host=<url-to-amq-broker-route>
```
[the remark this answers is not in the document]
Comment (mnovak1) [reply microprofile-reactive-messaging-amqp-connector-example/src/main/resources/META-INF/microprofile-config.properties:6]: Ok, I'll update it

**Comment by mnovak1** [comment]
@jmesnil All things should be fixed now.

File: microprofile-reactive-messaging-amqp-connector-example/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mnovak1) [reply microprofile-reactive-messaging-amqp-connector-example/README.md]: Done.

File: microprofile-reactive-messaging-amqp-connector-example/src/main/resources/META-INF/microprofile-config.properties (hunk 4 above)
[the remark this answers is not in the document]
Comment (mnovak1) [reply microprofile-reactive-messaging-amqp-connector-example/src/main/resources/META-INF/microprofile-config.properties]: Done

File: microprofile-reactive-messaging-amqp-connector-example/src/main/java/org/wildfly/examples/microprofile/reactive/messaging/amqp/connector/TestResource.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (mnovak1) [reply microprofile-reactive-messaging-amqp-connector-example/src/main/java/org/wildfly/examples/microprofile/reactive/messaging/amqp/connector/TestResource.java]: Done

**Comment by jmesnil** [comment]
@mnovak1 thanks!

**Pull request merged**